scripts/BuildAndTest.ps1

```diff
@@ -139,6 +139,8 @@ function New-NuGetPackageFromNuspecFile($project, $version, $suffix = "") {
     if ($LASTEXITCODE -ne 0) {
         Exit-WithFailureMessage $ScriptName "$project NuGet package creation failed."
     }
+
+    Write-Information "  Successfully created package '$BinRoot\NuGet\$Configuration\$Project.$version.nupkg'."
```

Comment:
`dotnet pack` produces this message for NuGet packages build from a project file. But we weren't producing any message for NuGet packages build from a .nuspec file. This message mimics the format (including the indentation) of the message from `dotnet pack`.